Makefile

```
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
+	docker compose -f functional_tests/docker-compose-dev-app.yml up -d
+
+.PHONY: functional-tests-down
+functional-tests-down:  ## Stop the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose-dev-app.yml down
+
+.PHONY: functional-tests-run
+functional-tests-run: collectstatic  ## Run the functional tests
+	DJANGO_SETTINGS_MODULE=cms.settings.functional_test poetry run ./manage.py migrate --noinput
```

As above, use Make commands where possible, so this could be replaced by:
```suggestion
	DJANGO_SETTINGS_MODULE=cms.settings.functional_test make migrate
```

---

I'm leaving this as a direct call so that I can supply the `--settings` option.

---

If that your preference over env var?. It is set as env var elsewhere?.

---

Just checked the docs and the command line flag has highest order of precedence, so for cases like this where we wouldn't want someone's accidentally left over `.env` settings to override it, it would be the best choice. I'll update the other uses to match 👍